index.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practices. Instead of disabling `eslint`, it will be nice to fix the issue.

Reply:
As mentioned in my previous comment.